packages/esm-patient-chart-app/src/encounter-list/hooks/usePatientDeathStatus.ts

```diff
@@ -0,0 +1,14 @@
+import { openmrsFetch, restBaseUrl } from '@openmrs/esm-framework';
+import useSWRImmutable from 'swr';
```

Comment:
nit, but this applies to all imports from `swr` in this PR.

```suggestion
import useSWR from 'swr';
```